web/src/components/DatePicker.tsx

```diff
@@ -1,37 +1,59 @@
-import { TextField } from '@material-ui/core'
 import React from 'react'
+import LuxonUtils from '@date-io/luxon'
+import { KeyboardDatePicker, MuiPickersUtilsProvider } from '@material-ui/pickers'
+import { PST_UTC_OFFSET } from 'utils/constants'
 
 interface DatePickerProps {
   testId?: string
   date: string
-  onChange: (d: string) => void
-  updateDate: () => void
+  updateDate: (d: string) => void
 }
 
 const DatePicker = (props: DatePickerProps) => {
   return (
-    <TextField
-      data-testid="date-of-interest-picker"
-      label="Date of Interest (PST-08:00)"
-      type="date"
-      value={props.date.slice(0, 10)} // 'YYYY-MM-DD'
-      variant="outlined"
-      onChange={e => {
-        const value = e.currentTarget.value
-
-        if (value) {
-          props.onChange(value)
-        }
-      }}
-      onSelect={props.updateDate}
-      onKeyDown={event => {
-        if (event.key === 'Enter') {
+    <MuiPickersUtilsProvider utils={LuxonUtils}>
+      <KeyboardDatePicker
+        label="Date of Interest (PST-08:00)"
+        value={props.date}
+        format="yyyy/MM/dd"
+        allowKeyboardControl={true}
+        InputAdornmentProps={{ position: 'start' }}
+        onAccept={d => {
+          if (d) {
+            const newDate = d
+              .setZone('UTC' + PST_UTC_OFFSET)
+              .toISO()
+              .slice(0, 10)
+            props.updateDate(newDate)
+            console.log('accept', newDate)
+          }
+        }}
+        onKeyDown={event => {
+          if (event.key === 'Enter') {
+            const newDate = event.currentTarget
+              //Gets the input component that holds the date value
+              .getElementsByTagName('input')[0]
+              .value.toString()
+              //Replaces the '/' in the date with '-' otherwise the formatting is incompatible
```

Comment:
Instead of this, can we do something like below for all the `on` methods:
```
const newDateString = event.currentTarget
              //Gets the input component that holds the date value
              .getElementsByTagName('input')[0]
              .value.toString()
              
const newDate = DateTime.from(newDateString).toISO()
```

This way we can leverage `luxon` to do the date formatting for us.